src/services/dataGenerator.js

```diff
@@ -0,0 +1,172 @@
+import faker from 'faker';
+
+const generateFakeData = (amount, dateData) => {
```

Comment:
what's this for? will it be removed afterwards?

Reply:
Yes it will be removed when API will be available.

Reply:
We are using it to mock some random data, by the example that you have given as in a [file](https://docs.google.com/document/d/1eKD_dnz7n8VPJIQAPqSzOaEU74Ct63GZmxFi7t-bBvE/edit?ts=5eb974d2#heading=h.8fljb4jw2vy).
If you don't mind there is no need to particularly close inspect this part of code because when API will be available we should remove all this code from this widget.

Reply:
OK, thanks for confirming. Let's ensure we remove it once you have the API integration.